src/ICSharpCode.SharpZipLib/Zip/ZipFile.cs

```diff
@@ -954,7 +978,7 @@ public Stream GetInputStream(long entryIndex)
 
 				case CompressionMethod.Deflated:
 					// No need to worry about ownership and closing as underlying stream close does nothing.
-					result = new InflaterInputStream(result, new Inflater(true));
+					result = new InflaterInputStream(result, InflaterPool.Instance.Rent(true));
```

Comment:
Again, to avoid surprise breakage on low-resource setups, perhaps make this depend on a public property, like `EnableInflatorPooling`?